### PR TITLE
[v7r0] JobState - restored logic of having global database clients

### PR DESCRIPTION
**Pull request by atsareg**
JobState object is instrumented to have database clients on the class level in order not to recreate them at each JobState object instantiation 

BEGINRELEASENOTES

*WMS
FIX: JobState - restored logic of having database clients at the class level

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Client/JobState/JobState.py [hunk 1]
```diff
@@ -18,12 +18,32 @@
 
 class JobState(object):
 
+  class DBHold:
```
Comment (chrisburr) [inline WorkloadManagementSystem/Client/JobState/JobState.py]: I guess using an old style class was accidental?
```suggestion
  class DBHold(object):
```

Comment (atsareg) [reply WorkloadManagementSystem/Client/JobState/JobState.py]: Updated

**Comment by fstagni** [comment]
Did you run the integration tests (e.g. in Jenkins) with this patch?

**Comment by atsareg** [comment]
No, I have tested it in my test v7r0 installation

**Pull request merged**